README.md

```diff
@@ -18,36 +18,35 @@ in AI can be evaluated on Unity’s rich environments and then made accessible
 to the wider research and game developer communities. 
 
 ## Features
-* Unity Engine flexibility and simplicity
-* Flexible single-agent and multi-agent support
-* Multiple visual observations (cameras)
-* Discrete and continuous action spaces
+* Unity environment control from Python
+* 10+ sample Unity environments
+* Support for multiple environment configurations and training scenarios
+* Train memory-enhanced Agents using deep reinforcement learning
```

Comment:
Maybe call out ODD?

Reply:
We do, see below "Flexible Agent control with On Demand Decision Making" - good enough or should we clarify?